scalableminds/webknossos
# Call wk-worker nuclei inferral job

- Adds a route that calls the webknossos-worker infer_nuclei job if jobsEnabled
- Shows the result of this job correctly in the job list
- Note that this PR does not yet include UI to trigger this route, that will be follow-up issue https://github.com/scalableminds/webknossos/issues/5631

### Steps to test:
- set up wk-worker locally
- visit http://localhost:9000/api/jobs/run/inferNuclei/sample_organization/l4_sample?layerName=color (exchange dataset name if necessary)
- job should show in job list
- result should be linked in job list after job is successful

### Issues:
- fixes #5555

------
